examples/website-demos/virtual-event/src/components/Chat/DMChannelList.tsx

```diff
@@ -125,23 +126,22 @@ export const DMChannelList = () => {
           </Window>
           <Thread />
         </Channel>
-      ) : (
-        <>
-          <ChannelList
-            EmptyStateIndicator={EmptyStateIndicators}
-            filters={filters}
-            List={ListUI}
-            options={options}
-            Preview={(props) => <PreviewUI {...props} setDmChannel={setDmChannel} />}
-            sort={sort}
-          />
-          <div className='start-chat'>
-            <div className='start-chat-button' onClick={() => setSearching(true)}>
-              Start a chat
-            </div>
-          </div>
-        </>
       )}
+      <div className={dmChannel && 'dm-hidden'}>
```

Comment:
this kinda thing should be done as a ternary otherwise you're passing `false` in as the class name